blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueMultiBranchPipeline.java

```diff
@@ -65,6 +65,11 @@
     public abstract BluePipelineContainer getBranches();
 
     /**
+     * @return Gives {@link BluePipelineContainer}
+     */
+    public abstract BluePipelineContainer getFindBranches(String type);
```

Comment:
@kzantow I believe it results in to findBranches API, to keep it restful we should avoid adding such APIs, this should really go as search API, see OmniSearch or maybe pass query parameter in getBranches() to return filtered collection of branches.

Maybe we do it as separate PR?